code/go/0chain.net/smartcontract/dbs/types.go

```diff
@@ -46,8 +46,8 @@ type StakePoolReward struct {
 	DelegateRewards map[string]currency.Coin `json:"delegate_rewards"`
 	// penalties delegate pools
 	DelegatePenalties map[string]currency.Coin `json:"delegate_penalties"`
-	// challenge id
-	ChallengeID string `json:"challenge_id"`
+	// allocation id
+	AllocationID string `json:"challenge_id"`
```

Comment:
Please use the same json tag name, the challenge id is different from the allocation id